### PR TITLE
feat(changelog): jobs-added-custom-ephemeral-storage-on-serverless-jobs 2024-03-20

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Thomas Tacquet

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+status: added
+author:
+    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-20
+category: serverless
+product: jobs
+---
+
+Serverless Jobs now offers the possibility to customise your custom ephemeral storage!
```
Comment (nerda-codes) [inline changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx:12]:
```suggestion
Serverless Jobs now offers the possibility of applying customized ephemeral storage! 
```

File: changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-20
+category: serverless
+product: jobs
+---
+
+Serverless Jobs now offers the possibility to customise your custom ephemeral storage! 
+
+Each Serverless Job has a limited storage a now you can customise it to allow more demanding use cases.
```
Comment (nerda-codes) [inline changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx:14]:
```suggestion
With custom ephemeral storage, your Serverless Jobs no longer have a limited storage, as you can customize them to allow more demanding use cases.
```

File: changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx (hunk 1 above)
Comment (SamyOubouaziz) [reply changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx:12]:
```suggestion
Serverless Jobs now offers the possibility to customise you ephemeral storage! 
```
cc @nerda-codes what do you think?

File: changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx (hunk 2 above)
Comment (SamyOubouaziz) [reply changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx:14]: Ephemeral storage is still limited to 10GB (used to be 5), so maybe
```suggestion
Each Serverless Job comes with an ephemeral storage capacity that you can now customize to accommodate for more demanding use cases.
```

Comment (SamyOubouaziz) [reply changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx:14]: cc @nerda-codes

**Review by jcirinosclwy — APPROVED** [review]

File: changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-20
+category: serverless
+product: jobs
+---
+
+Serverless Jobs now offers the possibility to customize your ephemeral storage! 
+
+Each Serverless Job comes with an ephemeral storage capacity that you can now customize to accommodate for more demanding use cases.
```
Comment (RoRoJ) [inline changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx:14]:
```suggestion
Each Serverless Job comes with an ephemeral storage capacity that you can now customize to accommodate more demanding use cases.
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**